lang/src/org/partiql/lang/syntax/LexerConstants.kt

```diff
@@ -585,7 +590,7 @@ internal const val DIGIT_CHARS = "0" + NON_ZERO_DIGIT_CHARS
 
 @JvmField internal val E_NOTATION_CHARS = allCase("E")
 
-internal const val NON_OVERLOADED_OPERATOR_CHARS = "^%=@+"
+internal const val NON_OVERLOADED_OPERATOR_CHARS = "^%=@+~"
 internal const val OPERATOR_CHARS = NON_OVERLOADED_OPERATOR_CHARS + "-*/<>|!"
```

Comment:
Hmm, just noticed the `OPERATOR_CHARS` variable isn't used anywhere. Can it be removed?